python/pipeline/experiment.py

```diff
@@ -333,6 +333,73 @@ def plot_calibration_curve(self):
         return fig
 
 
+@schema
+class SurgeryTypes(dj.Lookup):
+    definition = """ # Surgery types used in experiment.Surgery
+    surgery_type                : varchar(64)         # Types of surgery performed on mice
+    """
+    contents = [
+        ['Cranial Window and Headbar'],
+        ['Headbar'],
+        ['Burr Hole and Suture'],
+        ['C-Section'],
+        ['Viral Injection'],
+        ['Electroporation']
+    ]
+
+
+@schema
+class SurgeryOutcomes(dj.Lookup):
+    definition = """ # Surgery outcomes used in experiment.Surgery
+    surgery_outcome             : varchar(32)         # Possible outcomes of surgeries performed on mice
+    """
+    contents = [
+        ['Survival'],
+        ['Non-Survival'],
+    ]
+
+
+@schema
+class SurgeryQualities(dj.Lookup):
+    definition = """ # Surgery qualities used in experiment.Surgery
+    surgery_quality              : varchar(32)         # Possible qualities of surgeries performed on mice
+    """
+    contents = [
+        ['Good'],
+        ['Okay'],
+        ['Bad']
+    ]
+
+
+@schema
+class Surgery(dj.Manual):
+    definition = """ # List of surgeries performed on mice
+    -> mice.Mice
+    date                         : date                   # Date surgery was performed
+    ---
+    -> experiment.Person            
+    -> SurgeryOutcomes
+    -> SurgeryTypes
+    -> SurgeryQualities
+    ketoprofen = null            : decimal(4,3)           # Amount of Ketoprofen given to mouse
+    weight = null                : decimal(5,2) unsigned  # Weight of mouse before surgery
+    notes                        : varchar(256)           # Notes on surgery
+    """
+
+
+@schema
+class SurgeryStatus(dj.Manual):
+    definition = """ # Updates to surgeries performed on mice
+    -> Surgery
+    timestamp                           : timestamp              # Timestamp of entry
+    ---
+    euthanized = "False"                : enum("True","False")
+    day_one = "False"                   : enum("True","False")   # First day checkup performed
+    day_two = "False"                   : enum("True","False")   # Second day checkup performed
+    day_three = "False"                 : enum("True","False")   # Third day checkup performed
```

Comment:
If there are multiple checkups, then they belong in a different table, e.g. `SurgeryCheckup`

Reply:
There are three checkups that need to be performed. Every time the user modifies the surgery status, a newer entry is added with the values specified. All processing takes place on the newest entry. If checkups should become a new table, how would you suggest the table be structured?

Reply:
I still don't understand quite what you mean there. What I get is that they will sometimes checkup on the mouse surgery and optionally euthanize it. I would just have something like 
```python
@schema
class SurgeryStatus(dj.Manual):
    definition = """
    -> Surgery
    timestamp    :datetime
    ----
    euthanized    : boolean                           # was the mouse euthanized during this checkup
    """
```
that is the db friendly way.  You just insert a new entry for each checkup
If they wanna know how many checkups a mice has had, they'll have to check `len(SurgeryStatus & surgery_key)` or you can offer the `num_checkups` attribute via a property:

```python
@schema
class SurgeryStatus(dj.Computed ):
    definition = """.... """

    @property
    def num_checkups(self):
        return len(self)
# And people will call it like
(experiment.SurgeryStatus & key).num_checkups # returns a number
```
Alternatively, in SQL there's the concept of a View which is a virtual table with data computed on-the-fly from actual real tables (e.g.,  num_checkups computed from the number of rows in SurgeryStatus), we don't have that in python-datajoint but you could do something like:
```python
@schema
class SurgeryStatus(dj.Computed ):
definition = """....
"""
    def num_checkups():
        """ Number of checkups for each surgery in SurgeryStatus"""
        return SurgeryStatus.aggr(Surgery, num_checkups="COUNT(*)"
# And people will call it like
experiment.SurgeryStatus.num_checkups() # returns a table that has all primary keys of Surgery plus the attribute num_checkups
```

Reply:
Those three entries for checkups are there because everyone is required to check the mouse the next three days after a surgery. The reason I don't go off timestamps is because someone could check the mouse but forget to make a new entry that day.